packages/client/src/rpc/modules/debug.ts

```diff
@@ -457,4 +460,35 @@ export class Debug {
     this.client.config.logger.configure({ level: logLevels[level] })
     return `level: ${this.client.config.logger.level}`
   }
+
+  /**
+   * Sets the current head of the local chain by block number. Note, this is a
+   * destructive action and may severely damage your chain. Use with extreme
+   * caution.
+   * @param blockOpt Block number or tag to set as head of chain
+   */
+  async setHead(params: [string]) {
+    const [blockOpt] = params
+    if (blockOpt === 'pending') {
+      throw {
+        code: INVALID_PARAMS,
+        message: `"pending" is not supported`,
+      }
+    }
+
+    let headHash = await this.service.skeleton?.headHash()
+    const oldHead = headHash ? bytesToHex(headHash!) : undefined
+    const block = await getBlockByOption(blockOpt, this.chain)
+    try {
+      await this.service.skeleton?.setHead(block, true)
```

Comment:
This sets the skeleton head. Should the vm execution part also not be updated? :thinking:

Reply:
I've updated it to set the execution head as well. 🙂

Reply:
Cool, can you also add this check in the tests? :)

Reply:
I've updated the tests to include it.